.github/workflows/publish-release.yml

```diff
@@ -22,10 +22,19 @@ jobs:
           node-version: lts/*
           cache: npm
 
+      - name: Chrome — Publish Release
+        run: npx chrome-webstore-upload publish
```

Comment:
Typo: should be chrome-webstore-upload-cli (missing -cli)

Also I think a comment or note talking about how this is publishes the latest upload would be good to have for clarity of what the Action overall does

Reply:
Good point; this isn't obvious. Makes sense to put a link in the comment to the tool's documentation too.